### PR TITLE
Feat/#1

### PR DESCRIPTION
**Pull request by tobiaslocker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,5 +26,21 @@ git clone https://YOUR_USER_NAME@github.com/peh-empire/hello-github.git
 cd hello-github
 ```
 ## Add collaborators
-
 To manage access, move to [peh-empire's settings](https://github.com/peh-empire/hello-github/settings), then to `Manage access' and press `Invite a collaborator` and select by `username`
+## Create Issue
+Move to the repository at `github.com` and file a new Issue. Description shouldbe precise enough to work on task without further communication.
```
Comment (nsadigova) [inline README.md]: typo: should be

**Review by nsadigova — COMMENTED** [review]
please fix typo

**Review by nsadigova — CHANGES_REQUESTED** [review]
please fix typo2

**Comment by tobiaslocker** [comment]
@nsadigova Typo is fixed, can be merged

**Pull request merged**